common/client-libs/validator-client/src/validator_api/routes.rs

```diff
@@ -23,3 +23,4 @@ pub const SINCE_ARG: &str = "since";
 pub const STATUS: &str = "status";
 pub const REWARD_ESTIMATION: &str = "reward-estimation";
 pub const STAKE_SATURATION: &str = "stake-saturation";
+pub const INCLUSION_CHANCE: &str = "inclusion-prob";
```

Comment:
since it's an http route, would it make sense to just use the full name, i.e. `"inclusion-probability"`? what do people more familiar with web think? @mmsinclair @fmtabbara ?